src/main/webapp/dashboard.js

```diff
@@ -236,7 +235,7 @@ function createMenteeCardImage(imgSrc) {
  * Call functions to populate page sections with data.
  */
 function setUpPage() {
-  getUser(dummyGitHubID).then(createAboutMe);
+  getUser().then(createAboutMe);
```

Comment:
You need a catch block here in case the fetch request returns any error. Consider how you would like errors to be displayed to the user.